root/usr/share/cockpit/branding/nethserver/branding.css

```diff
@@ -220,3 +220,23 @@ input[type="checkbox"].form-control {
         grid-row: 3/4 !important;
     }
 }
+
+@media (min-width: 768px){
+.login-pf .container .login-area {
+    border-right: 1px solid #474747;
+    position: relative;
+    min-height: 1px;
+    padding-left: 0px;
+    padding-right: 14px;
+}
+}
+
+input[type="text"].form-control { 
+width: 90%;
+margin-left: auto;
+}
+input[type="password"].form-control {
+width: 90%;
+margin-left: auto;
+}
```

Comment:
For sure these selector are way too wide: you're going to change almost every form in the UI.
You should narrow the scope of the selectors.

Reply:
I am going to leave it without effect, what is happening is not css problems, by increasing the resolution of my screen it is possible to see well.

Reply:
it is not necessary to apply it for production